relayer/spy_relayer/src/listener/validation.ts

```diff
@@ -1,207 +1,5 @@
-import {
```

Comment:
This file will be removed and all of the types / functions will go into a utils.ts or something similar. Eventually they should make their way back into the sdk.